cmd/object-api-options.go

```diff
@@ -116,11 +116,21 @@ func getOpts(ctx context.Context, r *http.Request, bucket, object string) (Objec
 		}, nil
 	}
 
+	deletePrefix := false
+	if d := r.Header.Get("x-minio-delete-prefix"); d != "" {
```

Comment:
```suggestion
	if d := r.Header.Get("x-minio-force-delete"); d != "" {
```

We don't need to bring in new headers

Reply:
use xhttp.MinIOForceDelete